test/get-project-context.test.js

```diff
@@ -0,0 +1,171 @@
+import test from "ava";
+import getProjectContext from "../lib/get-project-context.js";
+
+test("Parse project path with https URL", (t) => {
+  t.is(
+    getProjectContext({ env: {} }, "https://gitlbab.com", "https://api.gitlab.com", "https://gitlab.com/owner/repo.git")
```

Comment:
First I thought this is a newly introduced typo, but it was already previously added in `get-repo-id.test.js`:
`gitlbab` => `gitlab`

Even though this is not the scope of this MR, but it also affects only this test file. Maybe you can find and replace the typo in this file 😄

Reply:
Sure, done :)